### ISSUE TITLE
Looking at Tesla Coil (and maybe Obelisk) can cause lagspikes

### ISSUE THREAD

**Issue by PlusGit** [PlusGit/theta#18]
I'm unsure if this is OpenGL only - needs testing.

**Comment by Proluiz** [comment]
Well i noticed on the FISRT TIME of you look to obelisk in opengl my framerte droped to 2, but after it i dont lose anymore fps both in speed and quality video mode. Note: Quality looking to obelisk make me lose only 1 frame per second.
![screenshot_doom_20150706_144851](https://cloud.githubusercontent.com/assets/6380147/8529150/92b172d2-23ee-11e5-8d2e-22d7f4989ba5.png)
![screenshot_doom_20150706_144942](https://cloud.githubusercontent.com/assets/6380147/8529151/92b7cf06-23ee-11e5-90c2-d612e54c3ad6.png)
![screenshot_doom_20150706_144847](https://cloud.githubusercontent.com/assets/6380147/8529152/92d4d074-23ee-11e5-9b63-509232502fc5.png)

**Comment by PlusGit** [comment]
seems it only occurs once, after the lagspikes are over... it's over, it doesn't happen again

**Comment by Proluiz** [comment]
yes, its something related to the momorys of computers, happen only in tesla also.
